tasks/scripts/build-release-notes

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+
+if [ -d version ]; then
+  version=v$(cat version/version)
+elif [ "$#" -eq 1 ]; then
+  version=$1
+else
+  echo "version must be specified, either as an argument or an input" >&2
+  exit 1
+fi
+
+echo "$version" > ./built-notes/release-name
+# if this fails, it's because we forgot to manually promote latest.md to $version.md
+cp concourse/release-notes/"$version".md ./built-notes/notes.md
```

Comment:
i think we can (and should) do this with an automated commit -- the constraint on whether a version is releasable should be controlled by `passed:` constraints in the pipeline, and something feels awkward to me about making a 'release commit' up front without first knowing whether the code in question is gonna pass... I can imagine some kind of deadlock where we think we can release, we promote the `latest.md` and then it turns out our code is broken, but by the time the next RC passes testing we've merged more features so our release notes have to change! and the cycle goes on.

However, the way it is here isn't gonna directly break anything, so let's save ^this work for another PR.

Reply:
yea, we were trying to come up with a process that encouraged a human to look over the contributor-submitted release notes as a whole and check that they're readable, but this workflow is clunky for sure. also tricky that we can't have the pipeline push directly to master and do this rename + push as part of the manually triggered release job from within the pipeline!

Reply:
maybe the testing jobs could first check to see if the latest commit is only a documentation change, and automatically pass in that case...